classes/migration/install/OMPMigration.inc.php

```diff
@@ -294,6 +327,12 @@ public function up(): void
             $table->string('currency_code_alpha', 3)->nullable();
             $table->string('payment_method_plugin_name', 80)->nullable();
         });
+
+        // Add doiId to submission files
+        Schema::table('submission_files', function (Blueprint $table) {
+            $table->bigInteger('doi_id')->nullable();
+            $table->foreign('doi_id')->references('doi_id')->on('dois')->nullOnDelete();
+        });
```

Comment:
Can this be added to `SubmissionFilesMigration`?

Reply:
Submission files only need the doi_id column in OMP. Is it better to keep this here because only OMP uses it or add it in pkp-lib so the `submission_files` table is consistent across applications?

Reply:
Ah, I see. Maybe I would move them up around other DOI-related migrations. But I know most of those will be moved to pkp-lib, so that may not be possible.

Reply:
Right, most of the DOI table set up has moved, but I've colocated the two OMP-specific DOI related things now.